pkgs/ffigen/test/code_generator_tests/expected_bindings/_expected_global_native_bindings.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+// ignore_for_file: type=lint
+import 'dart:ffi' as ffi;
+
+@ffi.Native<ffi.Int32>()
+external int test1;
+
+@ffi.Native<ffi.Pointer<ffi.Float>>()
+external final ffi.Pointer<ffi.Float> test2;
+
+@ffi.Array(10)
```

Comment:
We should probably use `@ffi.Array.multi([10])` here. Considering people might have array dimension greater than 5 (ffi.Array constructor limit), also it's more consistent with our [current usage inside Structs & Unions](https://github.com/dart-lang/native/blob/main/pkgs/ffigen/lib/src/code_generator/compound.dart#L136)

Reply:
Oh, right! I have moved the generation code for `@Array` annotations into a common method now shared between globals and structs.